ios/Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
-gem "fastlane"
+gem 'fastlane', :github => 'RishabhTayal/fastlane', :branch => 'testflight-reject-previous-build'   
```

Comment:
Wait for merge on testflight
- Issue: https://github.com/fastlane/fastlane/issues/12551
- PR: https://github.com/fastlane/fastlane/pull/12565